cpp/src/io/orc/stripe_data.cu

```diff
@@ -1284,7 +1285,10 @@ static __device__ void DecodeRowPositions(orcdec_state_s* s,
 
   if (t == 0) {
     if (s->chunk.skip_count != 0) {
-      s->u.rowdec.nz_count = min(min(s->chunk.skip_count, s->top.data.max_vals), blockDim.x);
+      s->u.rowdec.nz_count =
```

Comment:
there's a LOT of unsigned use in these kernels, I'd prefer not to bloat the scope of this PR to address all of these.

Reply:
I'm OK with multiple small PRs addressing this issue separately.